package.json

```diff
@@ -227,14 +240,49 @@
         }
       ],
       "@semantic-release/changelog",
-      [
-        "@semantic-release/npm",
-        {
-          "pkgRoot": "dist"
-        }
-      ],
+      "@semantic-release/npm",
       "@semantic-release/github",
       "@semantic-release/git"
     ]
-  }
+  },
+  "scripts": {
+    "clean": "aegir clean",
+    "lint": "aegir lint",
+    "build": "aegir build",
+    "release": "aegir release",
+    "test": "npm run lint && npm run test:node && npm run test:chrome && npm run test:ts",
+    "test:ts": "npm run test --prefix test/ts-use",
+    "test:node": "aegir test -t node --cov",
+    "test:chrome": "aegir test -t browser --cov",
+    "test:chrome-webworker": "aegir test -t webworker --no-cov",
+    "test:firefox": "aegir test -t browser -- --browser firefox --no-cov",
+    "test:firefox-webworker": "aegir test -t webworker -- --browser firefox --no-cov",
+    "test:electron-main": "aegir test -t electron-main"
+  },
+  "devDependencies": {
+    "@stablelib/sha256": "^1.0.1",
+    "@stablelib/sha512": "^1.0.1",
+    "@types/chai": "^4.3.0",
+    "@types/chai-as-promised": "^7.1.4",
+    "@types/mocha": "^9.0.0",
+    "@types/node": "^18.0.0",
+    "aegir": "^37.5.1",
+    "buffer": "^6.0.3",
+    "chai": "^4.3.4",
+    "chai-as-promised": "^7.1.1",
+    "cids": "^1.1.9"
+  },
+  "aegir": {
+    "build": {
+      "bundle": false
+    },
+    "test": {
+      "cov": true,
```

Comment:
```suggestion
```

Should we disable coverage being on by default and opt-into it in just the Node tests if the webworker tests aren't supporting it?

I had also experimented with not running webworker tests and just doing node and the browser. 🤔